main.go

```diff
@@ -25,14 +25,10 @@ func main() {
 	var vcenterURL string
 	var insecure bool
 
-	flag.StringVar(&vcenterURL, "vcenter-url", "", "URL for vcenter user:password@domain.com:port")
-	flag.Bool("insecure", true, "use an insecure connection")
+	flag.StringVar(&vcenterURL, "vcenter-url", "http://user:pass@127.0.0.1:8989/sdk", "URL for vCenter")
+	flag.Bool("insecure", false, "use an insecure connection to vCenter (default false)")
 	flag.Parse()
 
-	if len(vcenterURL) == 0 {
```

Comment:
Why was this removed?

Reply:
flag.StringVar sets a default value for it, unless overwritten (to "") by the user, which is unlikely. Even if the user overwrites it to "" it would be caught in the connection logic in newVCenterClient by govmomi.